tests/pfcwd/test_pfcwd_function.py

```diff
@@ -49,22 +49,22 @@ class PfcCmd(object):
     @staticmethod
     def isBufferInApplDb(asic):
         if not PfcCmd.buffer_model_initialized:
-            PfcCmd.buffer_model = asic.run_redis_cmd(
+            result = asic.run_redis_cmd(
                 argv=[
                     "redis-cli", "-n", "4", "hget",
                     "DEVICE_METADATA|localhost", "buffer_model"
                 ]
             )
-
+            if result:
+                PfcCmd.buffer_model = result[0]
             PfcCmd.buffer_model_initialized = True
             logger.info(
-                "Buffer model is {}, buffer tables will be fetched from {}".
-                    format(
+                "Buffer model is {}, buffer tables will be fetched from {}".format(
                     PfcCmd.buffer_model or "not defined",
                     "APPL_DB" if PfcCmd.buffer_model else "CONFIG_DB"
```

Comment:
This if condition will throw an error if 'result' is False. In 201911, we don't have 'buffer_model' in DEVICE_METADATA|localhost. Even in that case, we should read from config db

Reply:
Ignore. i see it defined in L45